src/main/scala/com.snowplowanalytics.snowplow.micro/Configuration.scala

```diff
@@ -51,17 +51,18 @@ object Configuration {
   object StorageConfig {
     case object None extends StorageConfig
     case object InMemory extends StorageConfig
-    case class Persistent(path: Path, ttl: Duration, cleanupInterval: Duration) extends StorageConfig
+    case class Persistent(path: Path, ttl: FiniteDuration, cleanupInterval: FiniteDuration) extends StorageConfig
   }
 
   object Cli {
     implicit val uriArgument: Argument[Uri] = Argument.from("uri") { str =>
       Uri.fromString(str).leftMap(_ => s"Invalid URI: $str").toValidatedNel
     }
 
-    implicit val durationArgument: Argument[Duration] = Argument.from("duration") { str =>
+    implicit val durationArgument: Argument[FiniteDuration] = Argument.from("duration") { str =>
```

Comment:
I changed this from `java.time.Duration` to `scala.concurrent.duration.FiniteDuration` because that's more consistent with what we use everywhere else in Snowplow.